tests/generate_onepsw_env_file.py

```diff
@@ -0,0 +1,17 @@
+import json
+import os
+import subprocess
+from pathlib import Path
+
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+
+subprocess.run(["op", "signin"])
+onepassword_entry = subprocess.check_output(
+    ["op", "item", "get", "dbt-coves-tests", "--format", "json"]
+)
+onepassword_entry = json.loads(onepassword_entry)
+dbt_coves_fields = onepassword_entry.get("fields")
+with open(os.path.join(BASEDIR, ".env"), "w+") as f:
```

Comment:
The file contents should be emptied before writing I think.

Reply:
@ssassi `w+` truncates the file: https://docs.python.org/3/library/functions.html#open